test/info_extraction/sample_cookbook_jl/NHosts1Vector.jl

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should include the expected output of running the script on this input file. So that the test can verify the output.

Reply:
We can use the standard [library](https://docs.julialang.org/en/v1/stdlib/Test/index.html. 
) unit testing for julia, it is powerful enough. 
Python3 includes a pretty powerful [stdlib](https://docs.python.org/3/library/unittest.html) for writing tests, we can probably get away with that, since there shouldn't be that much python code written, basically just scripts that we use to do some data munging if necessary.